File: src/common/api/common/error/ContactStoreError.ts
import { TutanotaError } from "@tutao/tutanota-error"

/**
 * Errors related to the native contact store/phone book.
 */
export class ContactStoreError extends TutanotaError {
	constructor(message: string) {
		super("ContactStoreError", message)
	}
}

File: app-ios/TutanotaSharedFramework/Contacts/ContactStoreError.swift
import Foundation

public struct ContactStoreError: TutanotaError {
	public static let name: String = "de.tutao.tutashared.ContactStoreError"
	public let message: String
	public let underlyingError: (any Error)?
	public init(message: String, underlyingError: (any Error)?) {
		self.message = message
		self.underlyingError = underlyingError
	}
}
